bpf/bpf_lxc.c

```diff
@@ -533,6 +533,7 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 	__be32 orig_dip;
 	__u32 __maybe_unused tunnel_endpoint = 0;
 	__u8 __maybe_unused encrypt_key = 0;
+	mac_t __maybe_unused vtep_mac = 0;
```

Comment:
can this move inside the block within `#ifdef ENABLE_VTAP`? wondering if we end up consuming 8 bytes of the stack, even if ENABLE_VTEP is not enabled as this is marked with `__maybe_unused`?

Reply:
I thought I have tried to  move vtep_mac within `#ifdef ENABLE_VTEP` and ran into some issue, but I can try again in case something I am missing.